src/getPublicKey.c

```diff
@@ -166,21 +167,32 @@ void handleGetPublicKey(uint8_t p1,
     // Read the key index from dataBuffer and set the genAddr flag according
     // to p2.
     ctx->keyIndex = U4LE(dataBuffer, 0);
-    ctx->genAddr = (p2 == P2_DISPLAY_ADDRESS);
 
-    // Prepare the approval screen, filling in the header and body text.
-    if (ctx->genAddr) {
-        memmove(ctx->typeStr, "Generate Address", 17);
+    if(p2 == P2_DISPLAY_NONE)
```

Comment:
```suggestion
    if (p2 == P2_DISPLAY_NONE)
```
So that general coding style is respected?